frontend/components/ChatMessages.tsx

```
@@ -0,0 +1,183 @@
+                )}
+                {message.type === 'orchestrator' && message.persona_name && (
+                  <Badge variant="secondary" className="text-xs">
+                    AI
+                  </Badge>
+                )}
+              </div>
+              <div className="text-sm whitespace-pre-wrap">
+                {message.text.split('\n').map((line, index) => {
+                  const boldFormatted = line.replace(/\*\*(.*?)\*\*/g, '<strong>$1</strong>')
+                  return (
+                    <div key={index} dangerouslySetInnerHTML={{ __html: boldFormatted }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)